src/test/ui/issues/issue-72253.stderr

```diff
@@ -0,0 +1,10 @@
+error: expected one of `.`, `;`, `?`, or an operator, found `,`
+  --> $DIR/issue-72253.rs:4:9
+   |
+LL |         .arg("1")
+   |                  - expected one of `.`, `;`, `?`, or an operator
```

Comment:
I'd like to see this line below the "^ unexpected token" line

Reply:
or as a separate help recommendation

Reply:
Why do you feel this is bad? This actually give you context as to why it was expecting a `;` or `?`, but it might be that I am just used to this way of displaying things.

Reply:
I wouldn't go so far as to say that it is bad.  I'm good with it if it is not simple to drop the message down so that they both point to the comma token.  This is definitely an improvement over the initial report.